src/extends/room/intel.js

```diff
@@ -239,7 +239,7 @@ Room.requestIntel = function (roomname) {
     Game.rooms[roomname].saveIntel()
     return
   }
-  if (!Game.map.isRoomAvailable(roomname)) {
+  if (!qlib.map.isRoomAvailable(roomname)) {
```

Comment:
```suggestion
  if (Game.map.getRoomStatus(roomname) !== 'closed') {
```
We can request intel from rooms outside our novice / respawn area